src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -216,97 +325,5461 @@ static ColliderJntSphInit D_80B7A888 = {
+        f32 dz = sOwnerHeadPos.z - sLurePos.z;
+
+        if ((sqrtf(SQ(dx) + SQ(dy) + SQ(dz)) < 25.0f) || (KREG(77) > 0)) {
+            KREG(77) = 0;
+            D_80B7A688 = 0;
+            D_80B7A68C = 1;
+            func_8010B680(globalCtx, 0x4087, NULL);
+        }
+    }
+
+    if (D_80B7A688 == 0) {
+        HIGH_SCORE(HS_FISHING) |= 0x1000;
+    } else if (D_80B7A688 == 1) {
+        HIGH_SCORE(HS_FISHING) &= ~0x1000;
+    }
+
+    if (KREG(77) < 0) {
+        KREG(77) = 0;
+        D_80B7A690 = 1;
+    }
+
+    if (D_80B7A67C != 0) {
+        D_80B7A67C--;
+        if (D_80B7A67C == 0) {
+            func_8010B680(globalCtx, D_80B7E086, NULL);
+        }
+    }
+
+    Fishing_HandleOwnerDialog(this, globalCtx);
+
+    D_80B7E14C = 0.0015f;
+    D_80B7A680++;
+
+    if ((D_80B7E0AC != 0) && D_80B7A674) {
+        Fishing_UpdateLure(this, globalCtx);
+    }
+
+    Fishing_UpdateEffects(globalCtx->specialEffects, globalCtx);
+    Fishing_UpdatePondProps(globalCtx);
+    Fishing_UpdateGroupFishes(globalCtx);
+
+    if ((D_80B7E0AC != 0) && (D_80B7A6CC == 0) && (player->actor.world.pos.z > 1360.0f) &&
+        (fabsf(player->actor.world.pos.x) < 25.0f)) {
+        player->actor.world.pos.z = 1360.0f;
+        player->actor.speedXZ = 0.0f;
+
+        if (D_80B7A6D0 == 0) {
+            D_80B7A6CC = 10;
+        }
+    }
+
+    if ((sSinkingLureLocation != 0) &&
+        (fabsf(player->actor.world.pos.x - sSinkingLureLocationPos[sSinkingLureLocation - 1].x) < 25.0f) &&
+        (fabsf(player->actor.world.pos.y - sSinkingLureLocationPos[sSinkingLureLocation - 1].y) < 10.0f) &&
+        (fabsf(player->actor.world.pos.z - sSinkingLureLocationPos[sSinkingLureLocation - 1].z) < 25.0f)) {
+        sSinkingLureLocation = 0;
+        D_80B7A6CC = 20;
+        func_800A9F6C(0.0f, 150, 10, 10);
+        func_80078884(NA_SE_SY_TRE_BOX_APPEAR);
+        Audio_SetBGM(0x101400FF);
+    }
+
+    if (KREG(0) != 0) {
+        KREG(0) = 0;
+        D_80B7E0B6 = 0;
+        D_80B7A6CC = 20;
+        func_800A9F6C(0.0f, 150, 10, 10);
+        func_80078884(NA_SE_SY_TRE_BOX_APPEAR);
+        Audio_SetBGM(0x101400FF);
+    }
+
+    if (D_80B7A6D0 != 0) {
+        D_80B7A6D0--;
+    }
+
+    switch (D_80B7A6CC) {
+        case 0:
+            break;
+
+        case 1: {
+            Camera* camera;
+
+            sCameraId = Gameplay_CreateSubCamera(globalCtx);
+            Gameplay_ChangeCameraStatus(globalCtx, MAIN_CAM, CAM_STAT_WAIT);
+            Gameplay_ChangeCameraStatus(globalCtx, sCameraId, CAM_STAT_ACTIVE);
+            camera = Gameplay_GetCamera(globalCtx, MAIN_CAM);
+            sCameraEye.x = camera->eye.x;
+            sCameraEye.y = camera->eye.y;
+            sCameraEye.z = camera->eye.z;
+            sCameraAt.x = camera->at.x;
+            sCameraAt.y = camera->at.y;
+            sCameraAt.z = camera->at.z;
+            D_80B7A6CC = 2;
+            Interface_ChangeAlpha(12);
+            D_80B7FECC = 0.0f;
+            // fallthrough
```

Comment:
```suggestion
```

Reply:
I'd usually agree in the case of a non matching since it can show that it was intended to have a fall through but in this case I think its obvious since it wouldn't match otherwise.

Reply:
Yeah, as Fig said these fallthrough comments are intentional and try to make it more obvious for future readers/users, so I think we should have them